doc/src/manual/strings.md

```diff
@@ -1142,7 +1142,7 @@ some confusion regarding the matter.
 
 Version numbers can easily be expressed with non-standard string literals of the form [`v"..."`](@ref @v_str).
 Version number literals create [`VersionNumber`](@ref) objects which follow the
-specifications of [semantic versioning](https://semver.org/),
+specifications of [semantic versioning](https://semver.org/spec/v2.0.0-rc.2.html),
```

Comment:
```suggestion
specifications of [semantic versioning 2.0.0-RC2](https://semver.org/spec/v2.0.0-rc.2.html),
```

I mean if someone reading it they don't see RC2

Reply:
it seems they use lowercase:
```suggestion
specifications of [semantic versioning 2.0.0-rc2](https://semver.org/spec/v2.0.0-rc.2.html),
```